commands/marketplace_purchase.go

```diff
@@ -52,24 +52,17 @@ func (c *marketplacePurchaseCmd) preRunE(cmd *cobra.Command, args []string) erro
 		}
 	}
 
-	pretty.Progress("Getting service data...", func() {
+	pretty.Progress("Getting offer data...", func() {
```

Comment:
It's still getting the offer data to display the price and duration in the confirmation prompt.